README.md

```diff
@@ -1,5 +1,13 @@
-# internship-dpki
-<b>Efficient Distributed Public Key Infrastructure (DPKI) for practical usage in Service Mesh security</b>
-Microservice architecture has taken the world of Cloud Computing by storm. It allows enterprises to save resources while having the potential to scale whenever required and build highly resilient and self-healing systems.
-However, the problem of service identity is still a very actual issue that historically relies on centralized Public Key Infrastructure (PKI). In this scenario, a central trusted authority distributes identities based on specific factors. If an adversary can confuse the central authority or take it over completely, the whole IT landscape is potentially compromised.
-Thanks to the latest advancements in blockchain technology and secure multiparty computation (a.k.a MPC), we believe we have the knowledge and the technology to do better.
+# Efficient Distributed Public Key Infrastructure (DPKI) for practical usage in Service Mesh security
+
+**temporary** *Alex Falzone, 2022*
+
+## Abstract:
+
+Microservice architecture has taken the world of Cloud Computing by storm. It allows enterprises to save resources while having the potential to scale whenever required and build highly resilient and self-healing systems. However, the problem of service identity is still a very actual issue that historically relies on centralized Public Key Infrastructure (PKI).
+In this scenario, a central trusted authority distributes identities based on specific factors. If an adversary can confuse the central authority or take it over completely, the whole IT landscape is potentially compromised. Thanks to the latest advancements in blockchain technology and secure multiparty computation (a.k.a MPC), we believe we have the knowledge and the technology to do better.
+
+
+## Table of contents:
+
+- [Chapter on outcome from DPKI academic research](https://github.com/YanchWare/internship-dpki/blob/main/academic_research.md)
```

Comment:
You are missing reference to the Reference section

Reply:
Why do I have to add the Bibliography section in the readme? In this file will not only go the links to the various files that I will create?
Of course, I think it's the same thing with the reference section.